.travis.yml

```diff
@@ -21,5 +21,6 @@ addons:
     - libfftw3-dev
 script:
     - nosetests -v --with-coverage --cover-package=chebpy
+    - CHEBPY_USE_MPL=0 CHEBPY_USE_PYFFTW=0 nosetests -v --with-coverage --cover-package=chebpy
```

Comment:
Interesting that the tests with `numpy.fft` fallback all pass...

Reply:
This is great work - thank you.

My retrospective justification for using pyfftw now seems more vague. It would probably be good to revisit this at some point, perhaps by tallying performance of each FFT engine with respect to a set of benchmark functions, say.

Which makes me think it might be good as a further enhancement for users to be able to switch FFT engines on demand, perhaps via UserPrefs?